<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupBox</class>
 <widget class="QGroupBox" name="GroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>506</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>165</width>
    <height>110</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Global Analysis Toolbox</string>
  </property>
  <property name="title">
   <string>Global Analysis Tools</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,5,0">
   <item row="2" column="0" colspan="3">
    <widget class="TableWidget" name="tableWidget"/>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="traceComboBox">
     <item>
      <property name="text">
       <string>No Trace Selected</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="measureButton">
     <property name="text">
      <string>Measure Tool</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="traceROICheck">
     <property name="text">
      <string>Show Trace ROI</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="logButton">
     <property name="text">
      <string>Log Data</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save Log</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QTextEdit" name="textEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;To use this toolbox, plot ROIs to a trace window and select the ROI you wish to analyze in the dropdown menu below. Use the Trace ROI to select a the time range, and log the data as you go.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableWidget</class>
   <extends>QTableWidget</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
